--- a/configure.ac
+++ b/configure.ac
@@ -2,7 +2,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.69])
-m4_define([release_version], m4_esyscmd_s([echo ${RELEASE_VERSION:-1.0.0-snapshot}]))
+m4_define([release_version], m4_esyscmd_s([echo ${RELEASE_VERSION:-3.5.1}]))
 AC_INIT([rtp2httpd],[release_version],[jsq2627@gmail.com],[rtp2httpd],[https://github.com/stackia/rtp2httpd])
 AC_CONFIG_SRCDIR([src/rtp2httpd.h])
 AC_CONFIG_HEADERS([src/config.h])
